test/examples/test_kissgp_additive_classification.py

```diff
@@ -30,10 +31,12 @@ def __init__(self):
         super(GPClassificationModel, self).__init__(
             grid_size=16, grid_bounds=[(-1, 1)], n_components=2
         )
-        self.mean_module = ConstantMean(constant_bounds=[-1e-5, 1e-5])
-        self.covar_module = RBFKernel(log_lengthscale_bounds=(-5, 6))
+        self.mean_module = ConstantMean(prior=SmoothedBoxPrior(-1e-5, 1e-5))
+        self.covar_module = RBFKernel(log_lengthscale_prior=SmoothedBoxPrior(-5, 6))
```

Comment:
Should this (and a few other instance) be `log_lengthscale_prior=SmoothedBoxPrior(exp(-5), exp(6), log_transform=True)`? Placing a uniform distribution directly over log space leads to a distribution over the actual parameter that is not uniform.